Source/PropertyTools.Wpf/Helpers/TypeHelper.cs

```diff
@@ -243,6 +243,9 @@ public static bool IsIListIList(Type type)
             }
 
             var listElementType = GetListElementType(type);
+            if (listElementType == null)

```

Comment:
A test similar to `TypeHelperTests.IsIListIList_ObservableCollectionObservableCollection_ReturnTrue`